includes/class-wc-stripe-customer.php

```diff
@@ -295,6 +297,17 @@ public function add_source( $source_id ) {
 							$wc_token->set_expiry_month( $response->card->exp_month );
 							$wc_token->set_expiry_year( $response->card->exp_year );
 						}
+
+						if ( 'source' === $response->object && 'three_d_secure' === $response->type ) {
```

Comment:
@diegocurbelo Just to let you know. I'll double test this again. I'm not sure how we were able to save a 3DS card before since the card info was in another object.

Also, this surely can be merged with the previous `if` if we do something like `$response->$response_type->prop_name`